### DISCUSSION TITLE
Compatibility with Electron Application

### DISCUSSION THREAD

**Discussion by kilian-bryon** [artilleryio/artillery#1836] [Artillery]
Hey all,

I was wondering if the Playwright engine could be used to launch an Electron Application instead of a standalone Chromium browser. I'm referring to the possibility of Playwright to use Electron: [https://playwright.dev/docs/api/class-electron](url)

Thanks!

**Comment by hassy** [comment]
hi @kilian-bryon 👋 interesting, I didn't know Playwright had support for Electron now! At the moment our integration supports Chromium only. It should be possible in theory, but would need to look into it

**Comment by kilian-bryon** [comment]
It's a bit of a weird combination because Electron is a desktop application. I'll look further to solve my problem!
